src/shapes/polyline.class.ts

```diff
@@ -41,11 +44,29 @@ import { makeBoundingBoxFromPoints } from '../util/misc/boundingBoxFromPoints';
        * @deprecated
        * @type Boolean
        * @default false
+       * @todo set default to true and remove flag and related logic
```

Comment:
we could in theory move the default to true, but not remove the flag totally unless we can benchmark that for large paths/polyline this code has zero performance impact

Reply:
Not in this pr. It will be a nightmare adjusting tests

Reply:
no i m saying in the future maybe we can switch it to true, but the flag will probably stay.

Reply:
> we could in theory move the default to true, but not remove the flag totally unless we can benchmark that for large paths/polyline this code has zero performance impact

This is part of my difficulty in communication.
I read this and understand you want it changed since you didn't explicitly write later, future whatever.